src/drawer.py

```diff
@@ -15,7 +19,9 @@ class Drawer:
     measurements into PDF coordinates.
     """
 
-    def __init__(self, trafo: Transformer, config) -> None:
+    def __init__(self, trafo: Transformer, config, regressionConfig) -> None:
+        self.regressionConfig = regressionConfig
```

Comment:
snake case, see [here](https://peps.python.org/pep-0008/#function-and-variable-names) and probably more useful [here](https://stackoverflow.com/a/8423697/9655481)

Reply:
Done